app/src/main/res/layout/edit_message_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id = "@+id/edit_message_view"
+    android:orientation = "horizontal">
+
+    <ImageView
+        android:id = "@+id/editImage"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:padding = "12dp"
+        android:src = "@drawable/ic_edit_24"
+        android:layout_gravity = "start|top"
+        app:tint="@color/grey_600">
+
+    </ImageView>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight = "1"
+        android:orientation = "vertical">
+        <TextView
+            android:id = "@+id/editMessageTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor = "@color/grey_600"
+            android:text = "@string/nc_edit_message_text">
+
+        </TextView>
+
+        <TextView
+            android:id = "@+id/editMessage"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:maxLines = "1"
+            tools:text = "Edit message very very very very very very very very very very long">
+        </TextView>
+
+    </LinearLayout>
+
+    <ImageView
```

Comment:
maybe we could align the "x" and check button vertically? so move the "x" a little bit to the left?

![grafik](https://github.com/nextcloud/talk-android/assets/2932790/450a2e53-ecbf-4369-9bfb-d2294c56a8a0)